core/src/main/scala/org/bitcoins/core/psbt/PSBT.scala

```diff
@@ -96,6 +101,10 @@ case class PSBT(
     PSBT(global, inputs, outputs)
   }
 
+  def combinePSBTAndValidate(other: PSBT): PSBT = {
```

Comment:
So we don't seem to actually use this anywhere? I guess that's the tough thing about creating a parallel method, we don't actually check anything anywhere because this is completely unused?

Reply:
We shouldn't need to use it because we aren't passing PSBTs outside the library for signing. We are safe because our sign function does the check

Reply:
>sign function does the check

Where? BIP143 is vulnerable itself

Reply:
The PSBT sign function checks that we are not vulnerable.

https://github.com/bitcoin-s/bitcoin-s/pull/2204/files#diff-0f7b76bc99b643f00b04d5214c76cd8bed7fa19db163b91a934f9872b00552b8R167